docs/configuration.md

```diff
@@ -38,9 +38,14 @@ The option `imageSwapPolicy` (default: `exists`) defines the mutation strategy u
 The option `imageCopyPolicy` (default: `delayed`) defines the image copy strategy used.
 
 * `delayed`: Submits the copy job to a process queue and moves on.
-* `immediate`: Submits the copy job to a process queue and waits for it to finish (deadline 8s).
-* `force`: Attempts to immediately copy the image (deadline 8s).
+* `immediate`: Submits the copy job to a process queue and waits for it to finish (deadline defined by `imageCopyDeadline`).
+* `force`: Attempts to immediately copy the image (deadline defined by `imageCopyDeadline`).
 
+## ImageCopyDeadline
+
+The option `imageCopyDeadline` (default: `8s`) defines the duration after which the image copy if aborted.
+
+This option only applies for `immediate` and `force` image copy strategies.
```

Comment:
Isn't the default `8` really?  Since the parameter is an integer of seconds?  It might also be worth noting that a value of `0` will be treated as if no value was provided (although that's pretty normal for Go projects, the user may already expect this).

Reply:
Regarding parsing of this field, if an integer is passed then it will be interpreted as a duration expressed in nanoseconds, while if a string with a correct format (see https://pkg.go.dev/time#ParseDuration) is passed, then it's interpreted using that unit. For instance `8s` is effectively converted to a duration of 8 seconds, just as `8000000000` would be too. 
Please let me know if you think we should be more explicit about this in the documentation.